snowplow_tracker/emitters.py

```diff
@@ -51,7 +51,7 @@ class Emitter(object):
     def __init__(
             self,
             endpoint: str,
-            protocol: HttpProtocol = "http",
+            protocol: HttpProtocol = "https",
             port: Optional[int] = None,
             method: Method = "get",
```

Comment:
Fyi, I think it would be good to move to POST as the default method, but I have a dilemma about the default buffer size that I think we should discuss a bit more. [Wrote it down in the issue here](https://github.com/snowplow/snowplow-python-tracker/issues/289).